lang/LangSource/SC_TerminalClient.cpp

```diff
@@ -298,8 +298,8 @@ static PyrSymbol * resolveMethodSymbol(bool silent)
 {
 	if (silent)
 		return s_interpretCmdLine;
-	else
-		return s_interpretPrintCmdLine;
+	
+	return s_interpretPrintCmdLine;
```

Comment:
i know this is no functional change, but it doesn't make the code exactly readable